internal/gatewayapi/testdata/httproute-with-mirror-filter-service-not-found.out.yaml

```diff
@@ -72,13 +72,13 @@ httpRoutes:
     parents:
     - conditions:
       - lastTransitionTime: null
-        message: 'Failed to process route rule 0: failed to validate the RequestMirror
-          filter: service default/service-unknown not found.'
-        reason: UnsupportedValue
-        status: "False"
+        message: Route is accepted
+        reason: Accepted
+        status: "True"
```

Comment:
The `Appcepted` status should be `True` when failed to Resolve the BackendRef for a MirrorFiliter.